src/input.js

```diff
@@ -21,14 +21,28 @@ export const mydata = () => {
   });
 };
 
+const totaltasks = [
+  {
+    description: 'clean the room',
+    completed: false,
+    index: 1,
+  },
+  {
+    description: 'do homework',
+    completed: false,
+    index: 2,
+  },
+];
+
 export const display = () => {
   const inner = document.querySelector('.show');
-  const totaltasks = create();
+  // const totaltasks = create();
```

Comment:
### [OPTIONAL]
Please remove the commented code to make your codebase looks clean and professional 👍